help_to_heat/frontdoor/consts.py

```diff
@@ -104,7 +108,7 @@
 supplier_field_shell = "Shell"
 supplier_field_utilita = "Utilita"
 supplier_field_utility_warehouse = "Utility Warehouse"
-supplier_fields = [
+supplier_fields_real = [
```

Comment:
Can this be made to read a bit more naturally? Would quibble that these aren't fields, they're values

Reply:
for what its worth most of these fields could be named as such, ie `supplier_field_utilitia` -> `supplier_field_value_utilita`. for now have renamed the `fields` to `field_values`